dev/com.ibm.ws.microprofile.config.1.3/bnd.bnd

```diff
@@ -0,0 +1,82 @@
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+javac.source: 1.8
+javac.target: 1.8
+
+Bundle-Name: com.ibm.ws.microprofile.config.1.3
+Bundle-SymbolicName: com.ibm.ws.microprofile.config.1.3
+
+Bundle-Description:MicroProfile Configuration 1.3 Implementation, version ${bVersion}
+
+
+-dsannotations-inherit: true
+-dsannotations: com.ibm.ws.microprofile.config13.archaius.Config13ProviderResolverImpl
+
+Export-Package: \
+    !com.ibm.ws.microprofile.config.resources, \
+    com.ibm.ws.microprofile.config.*, \
+    !com.ibm.ws.microprofile.config12.resources, \
+    com.ibm.ws.microprofile.config12.*, \
+    !com.ibm.ws.microprofile.config13.resources, \
+    com.ibm.ws.microprofile.config13.*
+
+Private-Package: com.ibm.ws.microprofile.config.resources, \
+                 com.ibm.ws.microprofile.config12.resources, \
+                 com.ibm.ws.microprofile.config13.resources
+
+Import-Package: \
+        org.eclipse.microprofile.config;version="[1.0,2)", \
+        org.eclipse.microprofile.config.spi;version="[1.2,2)", \
+        !com.ibm.ws.microprofile.config.*, \
+        *
+
+WS-TraceGroup: APPCONFIG
+
+Include-Resource: \
+  @${repo;com.ibm.ws.microprofile.config.1.2}!/!(META-INF/services|OSGI-INF)/*, \
+  META-INF/services=resources/META-INF/services
+
+Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
+
+-buildpath: \
+	com.ibm.ws.org.apache.commons.lang3.3.5;version=latest, \
+	com.ibm.ws.logging;version=latest, \
+	org.eclipse.osgi;version=latest, \
+	com.ibm.websphere.org.osgi.core;version=latest, \
+	com.ibm.websphere.org.osgi.service.component;version=latest, \
+	com.ibm.wsspi.org.osgi.service.component.annotations;version=latest, \
+        com.ibm.ws.microprofile.config.1.2;version=latest, \
+	com.ibm.websphere.org.eclipse.microprofile.config.1.3;version=latest, \
+	com.ibm.ws.kernel.service;version=latest, \
+	com.ibm.websphere.javaee.annotation.1.2;version=latest, \
+	com.ibm.ws.org.osgi.annotation.versioning;version=latest, \
+	com.ibm.websphere.org.osgi.service.cm;version=latest,\
+	com.ibm.ws.cdi.interfaces;version=latest,\
+	com.ibm.ws.container.service;version=latest,\
+	com.ibm.ws.app.manager;version=latest
+
+-testpath: \
+	../build.sharedResources/lib/junit/old/junit.jar;version=file, \
+	com.ibm.ws.junit.extensions;version=latest, \
+	org.hamcrest:hamcrest-all;version=1.3, \
+	org.jmock:jmock-junit4;strategy=exact;version=2.5.1, \
+	org.jmock:jmock;strategy=exact;version=2.5.1, \
+    org.glassfish:javax.json;strategy=exact;version=1.0.4, \
+    com.ibm.websphere.javaee.jsonp.1.0;version=latest, \
```

Comment:
Mixing tabs and spaces

Reply:
I use spaces ... except when I copy-paste from somewhere else!